catanatron_core/catanatron/models/actions.py

```diff
@@ -252,7 +252,7 @@ def maritime_trade_possibilities(state, color) -> List[Action]:
     hand_freqdeck = [
         player_num_resource_cards(state, color, resource) for resource in RESOURCES
     ]
-    port_resources = set(state.board.get_player_port_resources(color))
+    port_resources = state.board.get_player_port_resources(color)
```

Comment:
Nice! Redundant `set()` right?

Reply:
I think it's useful but I moved the set call to the method itself so it could be cached.